app/src/components/search/filters.tsx

```diff
@@ -52,12 +51,14 @@ const exampleMultiSelect = (name: string) => {
   );
 };
 
-const Filters = () => {
+const Filters = ({ isCollection }) => {
   const [isExpanded, setIsExpanded] = useState(false);
 
   return (
     <>
-      <Heading size="heading4">Refine your search</Heading>
+      <Heading size="heading4">
+        {`Refine your ${isCollection ? "results" : "search"}`}
```

Comment:
nitpick: this is fine, but my DS-pattern thinking wants me to add the entire heading text as a prop rather than have `Filters` "know" about whether it's rendering a collection set of filters or not.